packages/streaming-image-volume-loader/src/BaseStreamingImageVolume.ts

```diff
@@ -682,6 +798,44 @@ export default class BaseStreamingImageVolume extends ImageVolume {
       });
   }
 
+  /** Interleaves the values according to the stages definition */
+  public interleave(
```

Comment:
I was surprised to find this on the baseImageVolume. Is it meant for interleaving the imageIds? Our decision was to avoid including that logic in the ImageVolume and instead allow users to freely handle sorting, interleaving, and other actions as they wish.

https://www.cornerstonejs.org/live-examples/volumepriorityloading